pkgs/development/libraries/librealsense/default.nix

```diff
@@ -15,6 +20,16 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     libusb1
+    gcc.cc.lib
+  ] ++ lib.optional cudaSupport cudatoolkit
+    ++ lib.optionals enablePython (with pythonPackages; [
+      python
+      wheel
```

Comment:
Are wheel and setuptools still needed?

Reply:
True, they are no more useful, i removed them